Week-2/Day5/autoLoan/index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Auto Purchase Offer and Loan Comparison</title>
+  <link href="style.css" rel="stylesheet">
+</head>
+
+<body>
+  <div class="container">
+    <h1>Comparison of Dealer Incentives and Loan Offers</h1>
+    <div class="offer">
+      <div class="offerTitle">Purchase Offer 1</div>
+      <div class="offerContents">
+        <table>
+          <tr>
+            <th>Item</th>
+            <th>Cost</th>
+          </tr>
+          <tr>
+            <td>Purchase price:</td>
+            <td>$33,500</td>
+          </tr>
+          <tr>
+            <td>Cash incentive/rebate:</td>
+            <td>$0</td>
+          </tr>
+          <tr>
+            <td>Loan term (months):</td>
+            <td>48</td>
+          </tr>
+          <tr>
+            <td>Annual percentage rate (APR):</td>
```

Comment:
The APR should have a % sign within the input or as a label to clarify the expected format.